src/main/java/vazkii/botania/api/corporea/CorporeaHelper.java

```diff
@@ -167,69 +165,71 @@ public static int getCountInNetwork(ItemStack stack, Map<IInventory, Integer> in
 	 * The "matcher" parameter has to be an ItemStack or a String, if the first it'll check if the
 	 * two stacks are similar using the "checkNBT" parameter, else it'll check if the name of the item
 	 * equals or matches (case a regex is passed in) the matcher string.
+	 * <br><br>
+	 * When requesting counting of items, individual stacks may exceed maxStackSize for
+	 * purposes of counting huge amounts.
 	 */
-	public static List<ItemStack> requestItem(Object matcher, int itemCount, ICorporeaSpark spark, boolean checkNBT, boolean doit) {
-		List<ItemStack> stacks = new ArrayList();
+	public static List<ItemStack> requestItem(Object matcher, int itemCount, ICorporeaSpark spark, boolean checkNBT,
+			boolean doit) {
+		List<ItemStack> stacks = new ArrayList<ItemStack>();
 		CorporeaRequestEvent event = new CorporeaRequestEvent(matcher, itemCount, spark, checkNBT, doit);
-		if(MinecraftForge.EVENT_BUS.post(event))
+		if (MinecraftForge.EVENT_BUS.post(event))
```

Comment:
I have that imported, but clean up profile won't format the code (like those whitespaces between if and (). This is defined in Code Style = formatter.